scipy/sparse/_csr.py

```diff
@@ -122,6 +122,59 @@ def tobsr(self, blocksize=None, copy=True):
 
     tobsr.__doc__ = _spbase.tobsr.__doc__
 
+
+    def _broadcast_to(self, shape, copy=False):        
+        if self.shape == shape:
+            return self.copy() if copy else self
+        
+        shape = check_shape(shape, allow_nd=(self._allow_nd))
+
+        if np.broadcast_shapes(self.shape, shape) != shape:
+            raise ValueError("cannot be broadcast")
+        
+        if len(self.shape) == 1 and len(shape) == 1:
+            self.sum_duplicates()
+            if self.nnz == 0: # array has no non zero elements
+                return self.__class__(shape, dtype=self.dtype, copy=False)
+            
+            N = shape[0]
+            data = np.full(N, self.data[0])
+            indices = np.arange(0,N)
+            indptr = np.array([0, N])
+            return self.__class__((data, indices, indptr), shape=shape, copy=False)
```

Comment:
Make this CSR because 1D not supported by CSC and wont be. So only have to work with CSR
(`self._fmt_container` is either a `spmatrix` or a `sparray` with format "fmt")
```suggestion
            return self._csr_container((data, indices, indptr), shape=shape, copy=False)
```